examples/HierarchicalOptimization_examples/BachmannJakStat/models/BachmannJakStat_SHP1oe_syms.m

```diff
@@ -0,0 +1,159 @@
+function model = Bachmann_JAKSTAT_SHP1oe_syms()
+
+syms t
```

Comment:
we probably can do something like `model = Bachmann_JAKSTAT_syms()` and then simply apply necessary changes right?